src/Compilers/CSharp/Test/Semantic/Semantics/PropertyFieldKeywordTests.cs

```diff
@@ -1085,9 +1222,299 @@ .maxstack 8
 	}
 } // end of class C
 ");
+            Assert.Equal(1, accessorBindingData.NumberOfPerformedAccessorBinding);
+        }
+
+        [Fact]
+        public void TestAssigningFromConstructorNoAccessors()
+        {
+            var comp = CreateCompilation(@"
+public class C
+{
+    public C()
+    {
+        P = 5;
+    }
+
+    public int P { }
+
+    public static void Main()
+    {
+        System.Console.WriteLine(new C().P);
+    }
+}
+", options: TestOptions.DebugExe);
+            var accessorBindingData = new SourcePropertySymbolBase.AccessorBindingData();
+            comp.TestOnlyCompilationData = accessorBindingData;
+
+            Assert.Empty(comp.GetTypeByMetadataName("C").GetMembers().OfType<FieldSymbol>());
+            comp.VerifyDiagnostics(
+                // (6,9): error CS0200: Property or indexer 'C.P' cannot be assigned to -- it is read only
+                //         P = 5;
+                Diagnostic(ErrorCode.ERR_AssgReadonlyProp, "P").WithArguments("C.P").WithLocation(6, 9),
+                // (9,16): error CS0548: 'C.P': property or indexer must have at least one accessor
+                //     public int P { }
+                Diagnostic(ErrorCode.ERR_PropertyWithNoAccessors, "P").WithArguments("C.P").WithLocation(9, 16),
+                // (13,34): error CS0154: The property or indexer 'C.P' cannot be used in this context because it lacks the get accessor
+                //         System.Console.WriteLine(new C().P);
+                Diagnostic(ErrorCode.ERR_PropertyLacksGet, "new C().P").WithArguments("C.P").WithLocation(13, 34)
+            );
             Assert.Equal(0, accessorBindingData.NumberOfPerformedAccessorBinding);
         }
 
+        [Theory]
+        [InlineData("get;", 0, false)]
+        [InlineData("get; set;", 0, true)]
+        // [InlineData("set;")] PROTOTYPE(semi-auto-props): Not yet supported.
```

Comment:
>[InlineData("set;")] PROTOTYPE(semi-auto-props): Not yet supported. [](http://example.com/codeflow?start=11&length=67)

Is there a plan to support this scenario?

Reply:
Yes. But likely in a next PR (unless you think otherwise)